test/integration/targets/ansible-galaxy/setup.yml

```diff
@@ -1,11 +1,58 @@
 - hosts: localhost
+  vars:
+    ws_dir: /tmp/ansible-galaxy-webserver
   tasks:
-    - name: install git
+    - name: install git & OpenSSL
       package:
         name: git
       when: ansible_distribution not in ["MacOSX", "Alpine"]
       register: git_install
-    - name: save install result
+
+    - name: install OpenSSL
+      package:
+        name: openssl
+      when: ansible_distribution not in ["MacOSX", "Alpine"]
+      register: openssl_install
+
+    - name: install OpenSSL
+      apk:
```

Comment:
The `apk` module isn't in ansible-base, so you'll need to use the `command` or `shell` module to invoke the necessary package manager command(s) instead.

Reply:
Ah, thanks. Gave up waiting for the CI yesterday and didn't check back :-D Think this is now fixed in the latest push but I'm not overly familiar with Alpine/apk so if it's not looking quite correct please shout.